### PR TITLE
addedhelpful info

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5760

When previewing the changes made, i see only one "\" before Server FQDN, although i have written it correctly when making the change. I don't know if this is just a preview issue or not.

image

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 1]
```diff
@@ -325,6 +328,9 @@ Sign-in a workstation with access equivalent to a _domain user_.
 14. Click **Save**
 15. Sign-out of the Azure portal.
 
+> [!IMPORTANT]
+> For more details about the actual experience after everything is configured, please visit [this article](https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-how-it-works-authentication).
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md]:
```suggestion
> For more details about the actual experience after everything has been configured, please see [Windows Hello for Business and Authentication](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-how-it-works-authentication).
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 2]
```diff
@@ -154,6 +154,9 @@ These procedures configure NTFS and share permissions on the web server to allow
 ![CDP Share Permissions](images/aadj/cdp-share-permissions.png)
 9. In the **Advanced Sharing** dialog box, click **OK**.
 
+> [!Tip]
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md]:
```suggestion
> [!TIP]
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 3]
```diff
@@ -154,6 +154,9 @@ These procedures configure NTFS and share permissions on the web server to allow
...
+> Make sure users can access **\\Server FQDN\sharename**.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md]:
```suggestion
> Make sure that users can access **\\Server FQDN\sharename**.
```

Comment (illfated) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md]: Yes, that extra backslash as an escape character is necessary here.

**Comment by mypil** [comment]
@mapalko - PR has been copy edited and is ready for final review. This is based on issue #5760.

Thank you.

cc: @kenwith

**Comment by VLG17** [comment]
> Is it expected for there to be 3 "" in **\\Server FQDN\sharename**. I've only seen \ used before.

There should just be 2 of them "\\". Initially i was seeing just two when i modified it, as you can see in the printscreen from the top. I'll change it to 2.

**Comment by illfated** [comment]
> Is it expected for there to be 3 "" in **\\Server FQDN\sharename**. I've only seen \ used before.

The first \ works as an escape character. The following 2 backslashes are displayed.
MarkDown always read a directly leading backslash ( \ ) as an escape character.

Therefore, the first of the 3 \\\ is always required to display the next one.
(please notice that I have inserted 3 \\\ both here and above)

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 4]
```diff
@@ -154,6 +154,9 @@ These procedures configure NTFS and share permissions on the web server to allow
 ![CDP Share Permissions](images/aadj/cdp-share-permissions.png)
 9. In the **Advanced Sharing** dialog box, click **OK**.
 
+> [!Tip]
+> Make sure that users can access **\\Server FQDN\sharename**.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md:158]:
```suggestion
> Make sure that users can access **\\\Server FQDN\sharename**.
```

**Comment by illfated** [comment]
@mapalko : please notice what happens to backslashes in MarkDown here on GitHub, because in MarkDown, \\ is a special escape character. See also the comments with quotes from your comment. Those replies have all lost one of the backslashes because of the escape mechanism.

**Comment by illfated** [comment]
Just to be completely clear about this: I presume UNC paths with the hostname included should always start with 2 leading backslashes -- \\\HOSTNAME\FOLDER\PATH\SUB\
(again, note that I have put 3 \\\ to be able to show 2 \\ )

**Comment by illfated** [comment]
@VLG17 : Feel free to choose between reverting your latest commit or committing the suggestion I just posted.

**Comment by VLG17** [comment]
> @VLG17 : Feel free to choose between reverting your latest commit or committing the suggestion I just posted.

@illfated i commited the suggestion. Thank you for this, i've learned something new.

**Comment by illfated** [comment]
Mind you, there is one or two alternative methods to avoid the escape mechanism in MarkDown. In this case, I will say that it is mostly just one, but I will include both of them just to keep you informed:
1. Use MarkDown back ticks ( `` ) to isolate/encapsulate text to avoid HTML linking and MarkDown mechanisms being applied.
2. Use double quotation marks around text as a semi-encapsulation for some special characters.

E.g. (exempli gratia), with back ticks: **`\\Server FQDN\sharename`**
Withoud back ticks: **\\Server FQDN\sharename**

**Comment by mapalko** [comment]
@illfated Thanks for the very helpful explanation!

**Pull request merged**